app/views/orders/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1> Here are your past orders: </h1>
+
+<h1>  <% @orders.each do |order| %> </h1>
+<p> Your order from <%= order.resolved_at.to_s %> </p>  <!--- These could become links in story 13 --->
```

Comment:
Definitely like that idea. Nice use of comments here to let us know your thoughts!